.github/workflows/ci.yml

```diff
@@ -93,10 +93,24 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
         django-version:
           - "3.2"
           - "4.1"
           - "4.2"
+          - "5.0rc1"
+        exclude:
+          - django-version: "3.2"
+            python-version: "3.11"
+          - django-version: "3.2"
+            python-version: "3.12"
+          - django-version: "4.1"
+            python-version: "3.12"
+          - django-version: "5.0rc1"
```

Comment:
Since the [release is really close](https://code.djangoproject.com/wiki/Version5.0Roadmap), we should wait a couple of days to merge in order to declare the final version instead of the pre-release.